app/code/Magento/Quote/Model/Quote/Address.php

```diff
@@ -999,17 +999,24 @@ public function requestShippingRates(\Magento\Quote\Model\Quote\Item\AbstractIte
 
         $request->setFreeMethodWeight($item ? 0 : $this->getFreeMethodWeight());
 
+        
+         // =======================================================
```

Comment:
I don't think this prominent code fragment separation is necessary. Please remove extra lines and unnecessary comments before and after the changed code fragment to be consistent with the rest of the codebase.

Reply:
Hi Sivaschenko,
  Updating code as per ur suggession.

Thanks
Sheetal

Reply:
Hi @sheetal2jcommerce , sorry, I don't see the update. No new commits are added to the branch...